### DISCUSSION TITLE
刷牙的重要性，有刷牙不代表刷對！

### DISCUSSION THREAD

**Discussion by curtis1017** [1101-GS4719-C/Python-Community-GS4719-C#1240] [General]
一天中，我們至少刷兩次以上，比較在意牙齒整潔的人，一天刷牙的次數可能不止兩次，而是三餐後皆刷牙。但是刷牙不代表有用，在這篇文章中，我想要介紹一些刷牙的技術。

一、選擇刷頭小、刷毛軟的牙刷：軟毛刷頭可以減少對牙齒和牙齦的刺激，避免牙齦受傷，而刷頭較小可以讓牙刷更容易進入牙縫和狹窄的牙間隙，刷得更乾淨。
二、用45度斜角清潔：45度斜角刷牙，能讓刷毛更好地進入縫隙，清除掉那些難以清洗的食物殘渣，也可以避免過度刷牙造成牙齦受損。
三、內側清潔用牙刷後半部刷：口腔內側是大家常常刷不好的地方，如果使用牙刷前半部分刷內側，很容易因為角度不對刷不乾淨。用後半部來刷會讓牙刷更容易進到牙縫中，刷得更好。
四、使用含氟牙膏：研究證實氟化物可以有效預防蛀牙。氟化物可以干擾口腔細菌的代謝過程，減少蛀牙的發生。如果是6歲以下小朋友，每半年還可以到牙醫診所來免費塗氟，讓牙齒防禦力加倍。

我們可以透過以上四種方法，讓我們刷得更健康。

**Comment by zilinng** [comment]
我也覺得牙齒的健康真的非常重要，因此除了日常好好刷牙以外，我也會定期去診所檢查保養，如果牙齒出現問題便即時治療，另外，除了作者提到的刷牙小細節以外，我的牙醫也建議我使用牙線和電動牙刷，能夠讓我們日常的清潔更全面更有效率。

**Comment by CCello** [comment]
我自己是每天三餐飯後都會刷牙，但是沒有特別注意以上幾點，都是邊刷牙邊做其他事，不過下次開始我就會提醒自己有沒有做到以上的小技巧~

**Comment by KevinCheng42** [comment]
小時候去看牙醫時，醫生會要求我必須用雙手各刷一次，因為用不同手刷牙角度會不同，用雙手可以刷到的範圍也比較廣，可以彌補有些只有其中一隻手才刷得到的縫隙～

**Comment by Jack-1D** [comment]
我覺得這篇文章提供了一些實用的技巧，讓我們更有效地刷牙。尤其是要選擇刷頭小、刷毛軟的牙刷這個建議，這樣可以減少對牙齒和牙齦的傷害，同時也能刷得更乾淨。我也很贊同45度斜角清潔這個方法，可以更好地清除食物殘渣，避免過度刷牙對牙齦造成傷害。最後，使用含氟牙膏可以有效預防蛀牙，這也是很重要的一點。我會試著在日常生活中實踐這些技巧，讓我的口腔更健康。

**Comment by poweichen00** [comment]
除了刷牙技術之外，我非常推薦使用電動牙刷，不但本身有app能告訴你需要刷多久或是刷的力量太大會亮紅燈，這些都能幫助你更好瞭解刷牙的情況。

**Comment by Evening218** [comment]
口腔健康真的非常重要!另外我還想多補充一點:千萬不要覺得刷越多次牙越健康喔~我周遭真的很多人每天刷4~5次牙結果反而一大堆蛀牙QQ

**Comment by Chiuhsianglin** [comment]
國中的時候有做過一個檢測是要含某種液體，若有殘留可以表示你那個地方沒有刷乾淨，我就發現我牙齒上方，靠近牙齦那邊都會有殘留試劑，讓我明白我之後那邊要加強清潔。

**Comment by ziihui** [comment]
平常用牙線也很重要～也能更清潔牙齒間的縫隙！
造成蛀牙可能就是那些食物渣渣卡在縫隙沒清乾淨！

**Comment by harrycoconut** [comment]
我本身是一個很重衛生的人，除了用牙刷之外我還會搭配漱口水加強口腔清潔🧹從小到大我幾乎沒用過牙線，但神奇的是我從來沒有蛀牙過，我認為有很大一部分是搭配漱口水的關係！在這邊推薦給大家這個方法😌

**Comment by YoYooHuang** [comment]
我也覺得刷牙非常重要！但我曾經在電視上看過有一種口腔疾病是因為過度刷牙而引起，因為刷牙次數過多或是過於用力，導致牙齦受損，表面保護層被破壞而失去作用。所以在刷牙時，應該注意力道，以及各種刷牙技巧也得好好運用，不然刷牙反而害了自己！

**Comment by orianna901001** [comment]
我小時候因為害怕拔牙在牙醫診所裡大哭大鬧，甚至後來因為害怕蛀牙，所以我把四個智齒在高中時期全都拔完了，比較過一隻10元的牙刷跟百元起跳的牙刷，覺得整體來說還是便宜的很好用，自己也沒什麼蛀過牙，但是買過電動牙刷後還是十分有感，感覺靠近牙齦的部分也清潔了不少。

**Comment by Jamesmoneya** [comment]
文章中的第一點其實就是最著名的貝氏刷牙法，他的重點在於傾斜角度，可以輕微的刷到牙齦肉，將牙齦縫隙的髒汙清出，也不會太過讀刺激牙齦，但是要記得一次刷就刷兩顆，來回十次以上不要偷懶，以後如果蛀牙要抽神經，這樣的痛就有的受了

**Comment by hardtagottaname** [comment]
從前我也以為要吃完東西就要刷牙，但怎麼刷都還是容易蛀牙，直到我一次看牙醫時，牙醫才跟我說吃東西時會分泌酸性物質，若在這個時候刷牙，牙齒容易被腐蝕。要在吃完東西後15分鐘刷，讓口水稀釋掉酸性物質才不會對牙齒造成危害。因此，我同意筆者說的刷越多次牙不見得越好這個觀點，畢竟有些事物極必反，有效的刷牙才能對牙齒產生保護。

**Comment by llh26** [comment]
刷牙真的是一門學問，原本一直以為自己刷的很乾淨，結果用牙線才發現牙縫會有殘渣，那瞬間我直接懷疑人生，我之前的牙齒到底有多少殘渣沒弄掉⋯。用牙刷刷完真的要再用牙線處理，牙齒的健康才能維持更久～

**Comment by ashleyy-huang** [comment]
從小我看牙醫的次數就不可勝數，就算用了電動牙刷、牙線、漱口水，我還是會蛀牙，我覺得容不容易蛀牙跟基因有很大的關係，因為我的媽媽也和我一樣易蛀牙，可是我爸就不會有這個狀況，希望我可以不要再蛀牙了，不然抽神經做牙套好貴又好痛

**Comment by Superlan8** [comment]
我聽我認識的朋友，只要有用過電動牙刷的都說回不去，害我也想試一下，不然我每次看牙醫都說我刷牙太大力或是沒刷乾淨，最近來試試作者提供的方法，畢竟牙齒的健康真的很重要！

**Comment by RogelioKG** [comment]
我真的不懂。
為什麼有些人可以兩三天不刷，都沒蛀牙。
我真的是一天不刷就要蛀得要命。
如果世界上的有體質這玩意，那我就是易蛀體質。
後來想想，可能是因為小時候不太愛刷牙的緣故。
因為小時候蛀掉了，填上去的料強度注定是要比原本的牙齒更容易蛀掉。
現在也只好定期回診，真D無奈。

**Comment by Aaron0810** [comment]
我自己是通常一天刷一次而已，但有東西卡著或是吃比較重口味的東西才會多刷一次，不過我每次刷都會刷很久，其實活到現在確實也不知道到以怎樣的刷牙方式才是對的，10個牙醫可能有10種刷牙方式，但至少我現在這樣的刷牙方式來說是沒蛀牙的，所以或許只要不會蛀牙就是好的刷牙方式?
